_config.yml

```diff
@@ -111,6 +111,7 @@ icon-tag:
   galaxy-dropdown: fa-caret-down
   galaxy-wf-connection: fa-arrows-h
   galaxy-wf-new: fa-plus
+  galaxy-scratchbook: fa-th
```

Comment:
we already have this icon 4 lines above :)

Reply:
```suggestion
```